lumicks/pylake/detail/calibrated_images.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from skimage.measure import block_reduce
+
+
+class CalibratedKymographChannel:
+    """A kymograph image along with its calibration"""
+
+    def __init__(self, name, data, start, time_step, calibration, downsampling_factor=1):
```

Comment:
Yeah, I wasn't too happy about `calibration` either. Maybe `pixel_size`? Then it'd be consistent with `Kymo`.

Why switch the order of `name` and `data`? So that `name` can have a default? I'm a bit uncertain about this, since in the FD package we always put the `name` first (as mandatory arg).

Reply:
I had another look at it.

I'd prefer to keep this order in the constructor itself, since we have it that way in the other classes as well.

I renamed `calibration` to `pixel_size` since I agree with you that calibration is unnecessarily vague (since it could be defined both ways).